README.md

```diff
@@ -138,18 +138,24 @@ This plugin provides 2 commands
 
 This is the command the mappings call under the hood. You can use it directly
 and prefix it with a count to target a specific terminal. This function also takes
-a the `size` and `dir` as an argument e.g.
+arguments `size`, `dir` and `direction`. e.g.
 
 ```vim
-:ToggleTerm size=40 dir=~/Desktop
+:ToggleTerm size=40 dir=~/Desktop direction=horizontal
 ```
 
+If `dir` is specified on creation toggle term will open at the specified directory.
+
 The directory can also be specified as `git_dir` which toggleterm will then
 use to try and derive the git repo directory.
 *NOTE*: This currently will not work for work tree's or other more complex setups
 
-If specified on creation toggle term will open at the specified directory at the
-specified height.
+If `size` is specified on creation toggle term will open at the specified height/width
+(doesn't apply to direction=window/tab/float). If the terminal has been created before
```

Comment:
Can you remove the mention of `tab` in the possible directions here. It's fully unsupported atm, just something I toyed with implementing but don't want someone to see this and raise an issue about why it doesn't work since it's not expected to.